modules/asm/variables.tf

```diff
@@ -50,6 +50,20 @@ variable "channel" {
   default = ""
 }
 
+variable "mesh_management" {
+  default     = ""
```

Comment:
Might be nice to also support `null`.

Would need to bump the min TF version to `1.1`: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/modules/asm/versions.tf#L19

Reply:
Done!